MAINTAINERS.md

```diff
@@ -0,0 +1,59 @@
+# Maintainers
+
+This document contains list of present maintainers, their role and selection process.
+
+## Current Maintainers
+
+|   Username    |    Focus Area   |
+| ------------- | -------------   |
+|  MarcoFalke   | General, QA     |
+|  fanquake     | General, Build  |
+|  hebasto      | General, UI/UX  |
+|  achow101     | General, Wallet |
+|  glozow       | General, Mempool|
+
+## Responsibilities and roles
+
+Project maintainers have commit access and are responsible for merging patches
+from contributors. They perform a janitorial role merging patches that the team
+agrees should be merged. Maintainers make sure that patches are safe and align
+with the project goals. However, their decision is not the last step, as patches
+must also go through post-merge review and testing before being implemented.
+The maintainers’ role is by agreement of project contributors.
+
+## Self Merges
+
+In general, it is considered best practice for maintainers to avoid merging their
+own changes directly into the codebase. This is because merging your own code can
+create a potential conflict of interest, where a maintainer's personal interests
+may override the best interests of the project.
+
+However, it is acceptable for maintainers to merge their own changes if pull request
+has enough reviews and no other maintainer is available to merge.
+
+## Nomination and Selection of Maintainers
+
+The current maintainers or contributors can nominate a long term contributor
+in the [IRC channel](https://web.libera.chat/#bitcoin-core-dev) to take on the
+role of a maintainer. Contributors are free to self nominate as well.
+
+Maintainers are added by a new maintainer opening a pull-request to add their
+key to the Trusted Keys file. For example, see [this pull
+request](https://github.com/bitcoin/bitcoin/pull/23798). The adding of the key
+is reviewed by the community and merged by one of the maintainer. Before opening
```

Comment:
Typo: "maintainer" should be "maintainers"

Reply:
Fixed